nix/mobile/android/release.nix

```diff
@@ -1,25 +1,38 @@
 { stdenv, pkgs, deps, lib
 , androidPkgs, patchMavenSources, jsbundle, status-go }:
 
+let
+  inherit (lib) getEnvWithDefault;
```

Comment:
I don't think we gain much with this, and it just makes the structure harder to see.
Just call it as it is, with `lib.getEnvWithDefault`. Or, it might work to just get it directly in the arguments above.